src/mesh/interpolation.cxx

```diff
@@ -146,7 +146,7 @@ const Field3D interp_to(const Field3D &var, CELL_LOC loc, REGION region) {
           // coordinates
 
           Field3D var_fa = fieldmesh->toFieldAligned(var);
-          Field3D result_fa;
+          Field3D result_fa(fieldmesh);
```

Comment:
Not sure this is the best fix. This way it does not include the fix from L70 ...